src/client-store/key.proto

```diff
@@ -4,23 +4,14 @@ syntax = "proto3";
 package dvote.types.v1;
 
 message Key {
-    enum Type {
-        SECP256K1 = 0; // ETHEREUM
-        BABYJUBJUB = 1;
-    }
-    Type type = 1;
-
     // Human friendly seed phrase, derived with an HD path to produce a root private key
     // The root private key will still be uniquely derived for each entity
-    string encryptedMnemonic = 2;
-    
-    // Accouts have an N:N mapping with entities and private keys
-    string encryptedRootPrivateKey = 3;
-    string rootPublicKey = 4;
-    string rootAddress = 5;
+    string encryptedMnemonic = 1;
+    string path = 2;
+    string locale = 3;
+    string address = 4;
```

Comment:
could the address be a `bytes` field?

Reply:
Sure.

Reply:
Concerning on backing up the entire account you can add your opinion on my comment on Nate's question above.